### DISCUSSION TITLE
hosing jQuery.ui version a.13.1 on cdnjs.com

### DISCUSSION THREAD

**Discussion by HoomanBahreini** [cdnjs/cdnjs#14120] [General] [answered]
Lighthouse is showing me a warning, saying jQuery.ui version 1.12.1 has known vulnerabilities:

![image](https://user-images.githubusercontent.com/32623734/157218459-2ad5d875-2eaa-4889-be71-6b3098685a08.png)

I wanted to update the library to version 1.13.1, but seems like this version in not hosted on cdnjs.com

https://cdnjs.com/libraries/jqueryui

Would it be possible to include this version please?

**Comment by MattIPv4** [comment, marked as answer]
Please see https://github.com/cdnjs/packages/issues/1013 & https://github.com/cdnjs/packages/issues/889 -- we are unable to update jQueryUI at present as there is no distributable version available upstream for us to pull.
